dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -752,26 +771,30 @@ private void run(String[] argv) {
                     if (null != cmd) {
                         switch (cmd) {
                         case "-v":
-                        case "--version":
+                        case VERSION:
                             version();
                             break;
-                        case "--server-metadata":
+                        case SERVER_METADATA:
                             serverMetadata();
                             break;
-                        case "--upgrade":
+                        case UPGRADE:
                             upgrade("none");
                             break;
-                        case "--upgrade-stable":
+                        case UPGRADE_STABLE:
                             upgrade("stable");
                             break;
-                        case "--upgrade-unstable":
+                        case UPGRADE_UNSTABLE:
                             upgrade("unstable");
                             break;
-                        case "--clean-cache":
+                        case CLEAN_CACHE:
                             clean();
                             break;
                         default:
-                            invalid(cmd);
```

Comment:
Where the suggestion tool is run when the users command is of the form `dockstore INVALID_COMMAND`